crates/uv/tests/it/help.rs

```diff
@@ -115,6 +116,9 @@ fn help_flag() {
       version  Display uv's version
       help     Display documentation for a command
 
+    Options:
+      -V, --version  Print version
```

Comment:
I feel like the old description was better? This feels like a minor regression

Reply:
Unfortunately, it seems like you can't update the default version help message when using the clap macro approach. Do you think this regression is a reason to move this out of the release?

Reply:
Why do we need to use the clap macro? Can't we just make this flag only applicable at the root instead of global?

Reply:
See https://github.com/astral-sh/uv/pull/13108